not-to-release/sources/newsgroup/groups.google.com_FOOLED_1bf9cdc5a4c2ac48_ENG_20050904_130400.xml.conllu

```diff
@@ -1009,7 +1009,7 @@
 22	-	-	PUNCT	HYPH	_	25	punct	25:punct	SpaceAfter=No
 23	planet	planet	NOUN	NN	Number=Sing	25	compound	25:compound	SpaceAfter=No
 24	-	-	PUNCT	HYPH	_	25	punct	25:punct	SpaceAfter=No
-25	earth	earth	PROPN	NNP	Number=Sing	7	obl	7:obl:on	SpaceAfter=No
+25	earth	earth	NOUN	NN	Number=Sing	7	obl	7:obl:on	SpaceAfter=No
```

Comment:
Earth can be considered a PROPN I think (if talking about the planet)

Reply:
I was hesitating to retag "Earth" (as for "Moon" in another sentence). I guess you are right